lib/fcrepo_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def self.default_instance_options=(options)
   end
 
   def self.default_instance(options = {})
-    @default_instance ||= FcrepoWrapper::Instance.new default_instance_options.merge(options)
```

Comment:
In solr I wanted to be able to set what port the solr instance runs on. This allows me to  use solrwrapper just like I used to use jettywrapper.

Aka I run configure, and the run start and it hangs around in the background with a development and test instance for me to use.  Check out the use here: https://github.com/psu-stewardship/etda/blob/master/lib/tasks/solr.rake